src/ai/backend/manager/cli/__main__.py

```diff
@@ -302,21 +304,28 @@ async def _clear_terminated_sessions():
         async with connect_database(cli_ctx.local_config, isolation_level="AUTOCOMMIT") as db:
             async with db.begin() as conn:
                 log.info("Deleting old records...")
-                result = await conn.execute(
-                    sa.delete(kernels).where(kernels.c.terminated_at < expiration_date),
-                )
-                deleted_count = result.rowcount
+                result = (
+                    await conn.scalars(
+                        sa.select(SessionRow.id).where(SessionRow.terminated_at < expiration_date)
+                    )
+                ).all()
+                session_ids = cast(list[uuid.UUID], result)
+                if session_ids:
+                    await conn.execute(
+                        sa.delete(kernels).where(kernels.c.session_id.in_(session_ids))
+                    )
+                    await conn.execute(sa.delete(SessionRow).where(SessionRow.id.in_(session_ids)))
 
-                curs = await conn.execute(sa.select([sa.func.count()]).select_from(kernels))
+                curs = await conn.execute(sa.select([sa.func.count()]).select_from(SessionRow))
                 if ret := curs.fetchone():
                     table_size = ret[0]
                     log.info(
-                        "The number of rows of the `kernels` tables after cleanup: {}",
+                        "The number of rows of the `sessions` tables after cleanup: {}",
                         table_size,
                     )
         log.info(
             "Cleaned up {:,} database records older than {}.",
```

Comment:
As we have changed the implementation, I think this line should be also updated to reflect that we removed N "session records" instead of "database records".

Reply:
There is a info log telling the number of deleted records below

Reply:
L323 already states amount of deleted sessions so I think current implementation would be sufficient for users to understand what's under the hood.